ground/lib/parsers/aprs.js

```diff
@@ -0,0 +1,99 @@
+import _ from 'lodash';
+import assert from 'assert';
+import BufferOffset from 'buffer-offset';
+import crc32 from 'buffer-crc32';
+import Dissolve from 'dissolve';
+// import sprintf from 'sprintf'; //TODO - lat & lon format
+import { Transform } from 'stream';
+import util from 'util';
+
+//vanguard APRS protocol packets have fixed length data format
+export const MULTIMON_PREFIX_SIZE     = 26; //The Callsign and other data prefixed to the message content
+export const MSG_TYPE_LOCATION        = 1;
+export const MSG_TYPE_TELEMETRY       = 2;
+export const POSITION_DATA_LENGTH     = 43; 
+export const TELEMETRY_DATA_LENGTH    = 34;
+export const BASE_URL                 = ' https://www.google.com/maps?z=12&t=m&q=loc:';
+
+export class Parser extends Dissolve {
+  constructor() {
+    super();
+    this.loop(end => {
+      this.parse();
+    });
+  }
+
+  parse() {
+    this.string('begin', 1).tap(() => {
+      if (this.vars.begin === 'A') {
+        this.string('next',4).tap(() => {
+          if(this.vars.next === 'PRS:'){
+            this.parseHeader();
+          }
+        });
+      }
+    });
+  }
+
+  parseHeader(){
+    this.string('header', MULTIMON_PREFIX_SIZE);
+    this.string('type', 1).tap(()=> {
```

Comment:
nit: make sure your formatting is consistent. there is no space after the `()` here, but there is in other places (like line 27). same problem for your functions (i.e. `parseHeader(){` vs `parse() {`)